src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -819,25 +800,40 @@ LibraryServiceApi.prototype.updateBookIndex = function updateBookIndex() {
  *   projects/{project}/zones/{zone}/disks/{disk}.
  * @param {String} tag
  *   REQUIRED: The tag to add.
- * @param {?gax.CallOptions} options
+ * @param {gax.CallOptions=} options
  *   Overrides the default settings for this call, e.g, timeout,
  *   retries, etc.
- * @param {?APICallback<google.tagger.v1.AddTagResponse>} callback
+ * @param {APICallback<google.tagger.v1.AddTagResponse>=} callback
  *   The function which will be called with the result of the API call.
  * @returns {gax.EventEmitter} - the event emitter to handle the call
  *   status.
  * @throws an error if the RPC is aborted.
  */
 LibraryServiceApi.prototype.addTag = function addTag() {
   var args = arguejs({
-    'resource': String,
-    'tag': String,
-    'options': [gax.CallOptions],
-    'callback': [Function]
+    resource: String,
+    tag: String,
+    options: [gax.CallOptions],
+    callback: [Function]
   }, arguments);
   var req = {
-    'resource': args.resource,
-    'tag': args.tag
+    resource: args.resource,
+    tag: args.tag
   };
   return this._addTag(req, args.options, args.callback);
 };
+
+module.exports = function build(gaxGrpc) {
+  var grpcClient = gaxGrpc.load([{
+    root: require('google-proto-files')('..'),
+    file: 'library.proto'
+  }]);
+  var built = grpcClient.google.example.library.v1;
+
+  built.libraryServiceApi = function(opts) {
+    return new LibraryServiceApi(gaxGrpc, grpcClient, opts);
+  };
+  return built;
```

Comment:
So, `built` is the gRPC client, with an additional attribute, which is a builder for the GAX client?